package-lock.json

```diff
@@ -1,9 +1,10 @@
 {
-    "name": "foundry-lancer-weapon-fx",
+    "name": "lancer-weapon-fx",
```

Comment:
This is my fault: I have my repo cloned to a directory called `foundry-lancer-weapon-fx`. This change is good!

If we add this to the `package.json` it will ensure consistent naming in `package-lock.json` no matter how it is installed:

```json
"name": "lancer-weapon-fx",
```